tests/ci/ci_config.py

```diff
@@ -316,9 +316,12 @@
         "Integration tests flaky check (asan)": {
             "required_build": "package_asan",
         },
-        "Compatibility check": {
+        "Compatibility check (x86)": {
```

Comment:
it's amd64 in this file. Let's stick to it

Reply:
Okay, will fix.